bin/cqlsh.py

```diff
@@ -494,6 +494,8 @@ def __init__(self, hostname, port, color=False,
                     profiles[EXEC_PROFILE_DEFAULT].load_balancing_policy = WhiteListRoundRobinPolicy([self.hostname])
                 kwargs['port'] = self.port
                 kwargs['ssl_context'] = sslhandling.ssl_settings(hostname, CONFIG_FILE) if ssl else None
+                # workaround until driver would know not to lose the DNS names for `server_hostname`
+                kwargs['ssl_options'] = {'server_hostname': self.hostname} if ssl else None
```

Comment:
@margdoc do we have cases that using the unixsocket with SSL ? 

just wondering if the self.hostname in that case would work with the SSL as is, or we need to do something different

Reply:
> @margdoc do we have cases that using the unixsocket with SSL ?

I don't think so. Only the user on the same machine as the scylla node can connect by using unixsocket.
SSL is used in network communication.

Reply:
thanks